src/contents/encryption.rs

```diff
@@ -179,14 +128,13 @@ fn round_trip() -> Result<(), Error> {
 
 #[test]
 fn test_vector_1() -> Result<(), Error> {
```

Comment:
tests pass with existing test vectors, woooo :raised_hands:

Reply:
had to `#[should_panic]` those and used ChaCha20poly1305 ones in new test